tox.ini

```diff
@@ -94,7 +94,7 @@ commands =
     -a \
     -n \
     -W --keep-going \
-    -d "{toxinidir}/.tmp/.doctrees" \
+    -d "{toxinidir}/.tox/.doctrees" \
```

Comment:
This change is wrong, it's missing `.tmp`. Besides, it's unrelated, so submit it separately or wait for #543 to fix it.